apps/nextra/pages/en/build/standards/_meta.tsx

```diff
@@ -0,0 +1,9 @@
+export default {
+  index: "Introduction",
```

Comment:
I think we want to make the Standards directory in the sidebar clickable instead of doing this (https://developer-docs-nextra-git-greg-migrate-nextra-db9740-aptoslabs.vercel.app/en/build/standards). 

To do so, create a `standards.mdx` section under `/build` with the content from `index.mdx`